keyboards/eek/info.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please regenerate this file without the rotation, it currently looks like this in the Configurator:
![image](https://user-images.githubusercontent.com/4781841/100350958-d1bf1a80-303e-11eb-91aa-ba4c0c7d80f0.png)
(You can preview it by hitting Ctrl+Shift+I and selecting your info.json)

Reply:
Lol, wow. That’s pretty messed up. I’ll do that tonight!

Reply:
I believe I corrected this. :-)